README.md

```diff
@@ -372,10 +381,23 @@ endpoints:
         enabled: true
         description: "healthcheck failed"
         send-on-resolved: true
-```
 
+  - name: back-end
+    group: core
+    url: "https://example.org/"
+    interval: 5m
+    conditions:
+      - "[STATUS] == 200"
+      - "[CERTIFICATE_EXPIRATION] > 48h"
+    alerts:
+      - type: discord
+        enabled: true
+        description: "healthcheck failed"
+        send-on-resolved: true
```

Comment:
I don't think there's a need to explicitly showcase the effect that `alerting.*.overrides` has on the configuration, as it will create unnecessary noise in the documentation.

Reply:
https://github.com/TwiN/gatus/pull/271/commits/75c34d152d3adfa5cdb5e393ee45e8bdb3693b78